### DISCUSSION TITLE
"include" markdown resource from page bundle

### DISCUSSION THREAD

**Discussion by WizardUli** [McShelby/hugo-theme-relearn#956] [Q&A] [answered]
Hello. I have a normal page bundle, something like: `content/section1/page1/index.md` and a md resource `content/section1/page1/lala.md`.
Putting `{{% include "lala.md" %}}` into the `index.md` renders nothing.
Using `{{% include "content/section1/page1/lala.md" %}}` works but such hardcoding of paths kinda defeats the purpose of having page bundles.
Is including _resources_ not supported? Thanks.

**Comment by McShelby** [comment]
Works for me and in the exampleSite. Please post a link to your repo or a minimal reproucible example.

**Comment by WizardUli** [comment]
https://github.com/WizardUli/hugo-relearn-example-test

**Comment by McShelby** [comment, marked as answer]
Hugo reports the issue:

> WARN  Raw HTML omitted while rendering "C:/Daten/Soren/git/hugo-relearn-example-test/content/first-chapter/first-page/index.md"; see https://gohugo.io/getting-started/configuration-markup/#rendererunsafe

Add this to your `hugo.toml`

````
[markup.goldmark]
  renderer.unsafe = true
````

**Reply by WizardUli** [reply]
It works. Thank you.

But where does Hugo report such issue? I'm using
```
hugo server -D --disableFastRender --logLevel info
```
and see no such warning on stderr/stdout. I don't see it even in
```
rm -rf public/ && hugo --logLevel debug
```

**Reply by McShelby** [reply]
Glad, it worked for you!

> But where does Hugo report such issue? I'm using

This was a new feature for Hugo 0.139.
